### PR TITLE
feat: Added Navigation Component

### PR DESCRIPTION
**Pull request by ShekharDhangar**
Added Navigation component

### CODE REVIEW

File: components/Navigation/navigation-media-query.css [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+      .footer-container{
+         justify-items: flex-start;
+      }
+      .menu-list > li{  
+        line-height: 4.5rem;
+   }
+  }
+  @media only screen and (max-width: 610px) {
+      .sidebar{
+          min-width: 50vw !important;
```
Comment (partha8) [inline components/Navigation/navigation-media-query.css]: is !important required here?

**Review by partha8 — APPROVED** [review]
Looks good, and the navbar is also good. You haven't attached a click event to the hamburger icon right?

File: components/Navigation/navigation-media-query.css (hunk 1 above)
Comment (ShekharDhangar) [reply components/Navigation/navigation-media-query.css]: No, thanks for pointing out

Comment (ShekharDhangar) [reply components/Navigation/navigation-media-query.css]: No I haven't attached a click event

**Review by partha8 — APPROVED** [review]
Looks good

File: components/Navigation/navigation.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+const menuIcon = document.querySelector('.menu-btn');
+menuIcon.addEventListener('click', menuHandler);
+const sideBar = document.querySelector('.side-overlay');
+function menuHandler() {
+    console.log('clcikd')
+    sideBar.classList.add('open');
```
Comment (ApurvaSawant11) [inline components/Navigation/navigation.js:5]: Remove `console.log` if it's not needed.

File: components/Navigation/navigation.html [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+  </head>
+  <body>
+  <main>
+    <header>
+      <section class="flex nav-section ">
+        <div class="logo-container">
+          <a id="btnHamburger" href="#" class="header-toggle menu-btn icon">
+            <span></span>
+            <span></span>
+            <span></span>
```
Comment (vrx29) [inline components/Navigation/navigation.html:26]: There are multiple empty spans is it supposed to be like this??

File: components/Navigation/navigation.js (hunk 2 above)
Comment (ShekharDhangar) [reply components/Navigation/navigation.js:5]: Removed

**Review by ApurvaSawant11 — APPROVED** [review]
Looks good to me!

File: components/Navigation/navigation.html (hunk 3 above)
Comment (ShekharDhangar) [reply components/Navigation/navigation.html:26]: Yes, its for hamburger menu. I actually created it, not used icons

Comment (vrx29) [reply components/Navigation/navigation.html:26]: fine then you can merge.

**Review by vrx29 — APPROVED** [review]

**Pull request merged**